### ISSUE TITLE
There is no documentation for this project

### ISSUE THREAD

**Issue by grey9nr** [Shakyamuni177te/The-Order-of-the-Mouse#6]
Sort it out.

**Comment by Shakyamuni177te** [comment]
In progress. Refactoring.

**Comment by grey9nr** [comment]
I don't care if you're refactoring, just get it done.

**Comment by Shakyamuni177te** [comment]
I am running off Valve time at the moment.

**Comment by grey9nr** [comment]
You can't even bloody spell things right. And I don't want you negotiating with other companies either. Not that you'd do that properly.

**Comment by Shakyamuni177te** [comment]
Comment fixed. Happy now?

**Comment by grey9nr** [comment]
No. Listen, you need to pay proper attention when you code and write documentation. You can't just fire off any old thing that comes to mind and hope.

**Comment by Shakyamuni177te** [comment]
OK dude, that makes sense.

**Comment by grey9nr** [comment]
Don't get smart with me.

-----Original Message-----

**Comment by Shakyamuni177te** [comment]
Fuck off.

On 3 September 2015 at 08:26, grey9nr <notifications@github.com> wrote:

> Don't get smart with me.
>
>
> Grey Ninenr
> grey9nr@aim.com
>
>
>
>
> -----Original Message-----
> From: Peter David Carter <notifications@github.com>
> To: Shakyamuni177te/The-Order-of-the-Mouse <
> The-Order-of-the-Mouse@noreply.github.com>
> CC: grey9nr <grey9nr@aol.com>
> Sent: Wed, 2 Sep 2015 22:38
> Subject: Re: [The-Order-of-the-Mouse] There is no documentation for this
> project (#6)
>
>
>
> Comment fixed. Happy now?
>
>
> —
> Reply to this email directly or view it on GitHub.
>
>
>
>
>
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Shakyamuni177te/The-Order-of-the-Mouse/issues/6#issuecomment-137359324>
> .
>



-- 
Deer Simulator
https://www.facebook.com/pages/Deer-Simulator/1624237021193761
<https://github.com/Shakyamuni177te/The-Order-of-the-Mouse>
